docs/source/release_notes/release-notes.rst

```diff
@@ -32,7 +32,7 @@ Changelog
   :hidden:
 
 =======
-develop
+v0.61.3
```

Comment:
Lets add the release notes in a separate PR to develop too or it will never make it to the release-notes page.